tools/azure-pipelines/build-python-wheels.yml

```diff
@@ -14,15 +14,9 @@
 # limitations under the License.
 
 jobs:
-  - job: build_wheels
-    strategy:
-      matrix:
-        linux:
-          vm-label: 'ubuntu-20.04'
-        mac:
-          vm-label: 'macOS-10.15'
+  - job: build_wheels_on_Linux
     pool:
-      vmImage: $(vm-label)
+      vmImage: 'ubuntu-20.04'
```

Comment:
might be `'ubuntu-latest'`

Reply:
Not a fixed version, it may bring us potential problems in the future.